installer/Dockerfile.tmpl

```diff
@@ -1,4 +1,4 @@
-# syntax=docker.io/docker/dockerfile:1.4
+# syntax=docker/dockerfile:1.4
```

Comment:
Did you intend to make this change? I don't see the problem with the original line.